entity-types/ext-argocd/tests/Metric.stg.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "app_kubernetes_io_component": "applicationset-controller",
+    "app_kubernetes_io_instance": "argocd",
+    "app_kubernetes_io_managed_by": "Helm",
+    "app_kubernetes_io_name": "argocd-metrics",
+    "app_kubernetes_io_part_of": "argocd",
+    "app_kubernetes_io_version": "v3.0.12",
+    "argocd_kubectl_request_duration_seconds_count": {
+      "type": "cumulativeCount",
+      "count": 0,
+      "cumulative": 19
+    },
+    "cluster_name": "eks-prom-agent-1",
+    "collector.name": "prometheus-agent",
+    "collector.version": "2.1.0",
+    "endTimestamp": 1755622913415,
+    "entity.guid": "MTE5OTkwMzZ8RVhUfEFSR09DRHwzODc1Njg2Nzk3OTY1ODg3MDI0",
+    "entity.name": "argocd-metrics",
+    "entity.type": "ARGOCD",
```

Comment:
found it! if the telemetry already has the guid stamped our engine assumes it has already gone through entity synthesis so we just discard it as there is nothing to do here, I'd remove these three fields as they are stamped by synthesis service